app/models/item.rb

```diff
@@ -1,4 +1,8 @@
 class Item < ApplicationRecord
+  validates_presence_of :title
```

Comment:
Yeah no problem, good idea. I forgot you could do that with this syntax, I'm new to it